setup.rst

```diff
@@ -73,6 +73,22 @@ by pressing ``Ctrl+C`` from your terminal.
     You should **NEVER** listen to all interfaces on a computer that is
     directly accessible from the Internet.
 
+Storing your Project in git
+---------------------------
+
+Storing your project in git is easy! Just initialize you repository, add everything
```

Comment:
I'd like to keep the original references to GitHub and Bitbucket. "Git" is too abstract and it will help us when someone looks for "symfony github", etc.

```rst
Managing your project with `git`_ and storing it on services like
`GitHub`_ or `Bitbucket`_ is easy! ...

.. _`GitHub`: https://github.com/
.. _`Bitbucket`: https://bitbucket.org/
```